docs/user.md

```diff
@@ -0,0 +1,193 @@
+
+Here is a `MachineSet` YAML snippet on how to configure the label and taint as well:
+
+```yaml
+      metadata:
+        labels:
+          openshift-tests: "true"
+          node-role.kubernetes.io/tests: ""
+      taints:
+        - key: node-role.kubernetes.io/tests
+          effect: NoSchedule
+```
+
+
+### Privilege Requirements
+
+A user with [cluster administrator privilege](https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html#creating-cluster-admin_using-rbac) must be used with the provider certification tool. You also use the default kubeadmin user if you wish. 
+
+
+## Install
+
+
+### Prebuilt Binary
+
+The provider certification tool is shipped as a single executable binary which can be downloaded from:
+
+[https://github.com/redhat-openshift-ecosystem/provider-certification-tool](https://github.com/redhat-openshift-ecosystem/provider-certification-tool)
+
+The provider certification tool can be used from any system with network access to the OpenShift cluster under test. 
+
+
+### Build from Source
+
+See the development guide for more information.
+*TODO - link*
+
+
+## Usage
+
+
+### Run provider certification tests
+
+```sh
+openshift-provider-cert run 
+openshift-provider-cert run -w # Keep watch open until completion
+```
+
+
+### Check status
+
+```sh
+openshift-provider-cert status 
+openshift-provider-cert status -w # Keep watch open until completion
+```
+
+
+### Retrieve the results
+
+The certification results must be retrieved from the OpenShift cluster under test using:
+
+```sh
+openshift-provider-cert retrieve
+openshift-provider-cert retrieve ./destination-dir/
+```
+
+You can see a summarized view of the results using:
+
+```sh
+openshift-provider-cert results retrieved-archive.tar.gz
+```
+
+
+### Environment Cleanup
+
+Once the certification process is complete and you are comfortable with destroying the test environment:
+
+```sh
+openshift-provider-cert destroy
+```
+
+You will need to destroy the OpenShift cluster under test separately. 
+
+
+### Certification Failures
+
+If you already know the reason for a certification failure then resolve the problem and re-run the provider certification tool again so a new certification archive is created. 
+
+If you are not sure why you have failed certification or if some of the tests fail intermittently, proceed with the troubleshooting steps below. 
+
+
+### Troubleshooting
+
+#### Information Gathering
+
+Using the _status_ command will provide a high level overview but more information is needed to troubleshoot a problem with the provider certification tool or why you aren’t able to pass all certification checks. A Must Gather from the cluster and Inspection of sonobuoy namespace is the best way to start troubleshooting:
```

Comment:
> is the best way to start troubleshooting

Not sure about that. must-gather could be more complex than the sonobuoy's tarball. Let me write one section about the sonobuoy tarballs inspect.